app/src/main/java/com/p2p/presentation/tuttifrutti/TuttiFruttiActivity.kt

```diff
@@ -11,12 +11,17 @@ class TuttiFruttiActivity : BaseActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         if (savedInstanceState == null) {
-            addFragment(CreateTuttiFruttiFragment.newInstance(), shouldAddToBackStack = true)
+            addFragment(
+                CreateTuttiFruttiFragment.newInstance(instructions),
+                shouldAddToBackStack = true
+            )
         }
     }
 
     companion object {
-        fun start(context: Context) {
+        lateinit var instructions: String
```

Comment:
![](https://media0.giphy.com/media/3WmWdBzqveXaE/giphy.gif)

Esto no hay que hacerlo así. Si querés pasar data entre activitys tenés que usar los intents: https://www.geeksforgeeks.org/android-how-to-send-data-from-one-activity-to-second-activity/.

Igualmente, como decía Vale arriba, no entiendo por qué el tutti frutti no conocería sus propias instrucciones. Me parece muy piola que las traiga de un source, pero por qué no las trae el view model de tutti frutti directamente?

Reply:
No entiendo como hacerlo para q levante una sola vez, si lo pongo en el del tuti frutti